### PR TITLE
Add RPCS3

### PR DESCRIPTION
**Pull request by hcorion**

### CODE REVIEW

File: communities.json [hunk 1]
```diff
@@ -125,5 +125,12 @@
       "inviteCode": "obsproject",
       "githubUrl": "https://github.com/obsproject/obs-studio"
     }
+    {
+      "logo": "rpcs3.svg",
+      "title": "RPCS3",
+      "quote": "RPCS3 is the world's first PlayStation 3 emulator. We use Discord to provide support for our users, has allowed easy collaboration between contributors and is a great place for our patrons to discuss development.",
```
Comment (Megamouse) [inline communities.json]: read that again, please

Comment (MrHulaHoop) [reply communities.json]: Maybe reword to something like "*We use Discord to provide support for our users, collaborate with contributors and discuss development with our patrons.*"

**Comment by jfurrow** [comment]
@hcorion This invite link has expired; could you please submit another PR with a permanent invite link?
